### DISCUSSION TITLE
Install in python virtual environment

### DISCUSSION THREAD

**Discussion by r3tner** [davidhi7/ddcci-plasmoid#78] [General]
Installing through `pipx` gave me an error in the widget `module name not found`.

I could only make it work by installing the module into a virtual environment and modifying the command in the widget:
`$HOME/.venvs/my_virtual_environment/bin/python3 -m ddcci_plasmoid_backend`

**Comment by davidhi7** [comment]
pipx also essentially just installs the package into a simple virtual environments, so it should work as well assuming the widget backend command is `~/.local/bin/ddcci_plasmoid_backend`.
